test/system/lifecycle/lifecycle_test.go

```diff
@@ -97,14 +95,11 @@ var _ = Describe("Shoot oidc service testing", func() {
 		}
 
 		By("Verify that PodDisruptionBudget is correctly configured")
-		seedk8sVersion, err := semver.NewVersion(f.SeedClient.Version())
 		Expect(err).NotTo(HaveOccurred())
 
```

Comment:
This could be removed. No need to check `err` after we removed the line that could return error.

```suggestion
```